[ERROR] 2024-03-10 21:03:21,378 com.targetindia.day_3.programs.assignment_1.CustomerDatabase com.targetindia.day_3.exceptions.InvalidEmailException: Email is not valid
[ERROR] 2024-03-10 21:10:53,537 com.targetindia.day_3.programs.assignment_1.CustomerDatabase com.targetindia.day_3.exceptions.InvalidEmailException: Email is not valid
[ERROR] 2024-03-10 21:11:05,932 com.targetindia.day_3.programs.assignment_1.CustomerDatabase java.lang.NumberFormatException: For input string: "`"
[ERROR] 2024-03-10 21:11:20,016 com.targetindia.day_3.programs.assignment_1.CustomerDatabase com.targetindia.day_3.exceptions.InvalidPhoneException: Invalid Phone Number
[TRACE] 2024-04-06 14:50:13,953 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-06 15:24:13,153 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-06 15:24:15,936 com.targetindia.miniproject.ui.Main user made a choice 1
[TRACE] 2024-04-06 15:24:46,489 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-06 15:24:48,417 com.targetindia.miniproject.ui.Main user made a choice 1
[TRACE] 2024-04-06 15:24:51,408 com.targetindia.miniproject.ui.Main user made a choice 3
[TRACE] 2024-04-06 15:25:07,504 com.targetindia.miniproject.ui.Main user made a choice 4
[TRACE] 2024-04-06 16:42:53,085 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-06 16:42:55,233 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-06 16:42:56,022 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[TRACE] 2024-04-10 12:24:20,935 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-10 12:24:23,150 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-10 12:24:23,904 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[ WARN] 2024-04-10 12:24:25,421 org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator HHH000342: Could not obtain connection to query metadata
java.lang.IllegalStateException: Cannot get a connection as the driver manager is not properly initialized
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.getConnection(DriverManagerConnectionProviderImpl.java:262)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcEnvironmentInitiator.java:316)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:152)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:34)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:119)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:264)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:239)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:216)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:45)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:125)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:248)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:216)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.convertSqlException(BasicConnectionCreator.java:121)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:43)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:61)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections.addConnections(DriverManagerConnectionProviderImpl.java:501)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections.<init>(DriverManagerConnectionProviderImpl.java:367)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections$Builder.build(DriverManagerConnectionProviderImpl.java:552)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:105)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:81)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:125)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:248)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:216)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:283)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:150)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:34)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:119)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:264)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:239)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:216)
	at org.hibernate.boot.model.relational.Database.<init>(Database.java:45)
	at org.hibernate.boot.internal.InFlightMetadataCollectorImpl.getDatabase(InFlightMetadataCollectorImpl.java:230)
	at org.hibernate.boot.internal.InFlightMetadataCollectorImpl.<init>(InFlightMetadataCollectorImpl.java:198)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:166)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:1380)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1451)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at com.targetindia.miniproject.utils.JpaUtil.<clinit>(JpaUtil.java:14)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:59)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
[ WARN] 2024-04-10 12:24:25,495 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 90067, SQLState: 90067
[ERROR] 2024-04-10 12:24:25,495 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Connection is broken: "java.net.ConnectException: Connection refused: no further information: localhost" [90067-224]
[TRACE] 2024-04-10 12:36:14,263 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-10 12:36:16,474 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-10 12:36:16,794 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[ WARN] 2024-04-10 12:36:16,837 org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator HHH000342: Could not obtain connection to query metadata
java.lang.IllegalStateException: Cannot get a connection as the driver manager is not properly initialized
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.getConnection(DriverManagerConnectionProviderImpl.java:262)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcEnvironmentInitiator.java:316)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:152)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:34)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:119)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:264)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:239)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:216)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:45)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:125)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:248)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:216)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.convertSqlException(BasicConnectionCreator.java:121)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:43)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:61)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections.addConnections(DriverManagerConnectionProviderImpl.java:501)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections.<init>(DriverManagerConnectionProviderImpl.java:367)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections$Builder.build(DriverManagerConnectionProviderImpl.java:552)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:105)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:81)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:125)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:248)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:216)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:283)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:150)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:34)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:119)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:264)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:239)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:216)
	at org.hibernate.boot.model.relational.Database.<init>(Database.java:45)
	at org.hibernate.boot.internal.InFlightMetadataCollectorImpl.getDatabase(InFlightMetadataCollectorImpl.java:230)
	at org.hibernate.boot.internal.InFlightMetadataCollectorImpl.<init>(InFlightMetadataCollectorImpl.java:198)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:166)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:1380)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1451)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at com.targetindia.miniproject.utils.JpaUtil.<clinit>(JpaUtil.java:14)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:59)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
[ WARN] 2024-04-10 12:36:16,860 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 90067, SQLState: 90067
[ERROR] 2024-04-10 12:36:16,860 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Connection is broken: "java.net.SocketException: An established connection was aborted by the software in your host machine: localhost" [90067-224]
[TRACE] 2024-04-10 12:37:07,524 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-10 12:37:10,940 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-10 12:37:11,260 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[ WARN] 2024-04-10 12:37:12,108 org.hibernate.tool.schema.internal.ExceptionHandlerLoggedImpl GenerationTarget encountered exception accepting command : Error executing DDL "
    alter table if exists customers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "
    alter table if exists customers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:92)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlString(AbstractSchemaMigrator.java:574)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlStrings(AbstractSchemaMigrator.java:514)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.migrateTable(AbstractSchemaMigrator.java:333)
	at org.hibernate.tool.schema.internal.GroupedSchemaMigratorImpl.performTablesMigration(GroupedSchemaMigratorImpl.java:84)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.performMigration(AbstractSchemaMigrator.java:232)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:117)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:284)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:143)
	at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:140)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:37)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:292)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:431)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1455)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at com.targetindia.miniproject.utils.JpaUtil.<clinit>(JpaUtil.java:14)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:59)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: NULL not allowed for column "ID"; SQL statement:
INSERT INTO "PUBLIC"."CUSTOMERS_COPY_5_0"("CUSTOMER_ID", "COMPANY_NAME", "CONTACT_NAME", "CONTACT_TITLE", "ADDRESS", "CITY", "REGION", "POSTAL_CODE", "COUNTRY", "PHONE", "FAX") OVERRIDING SYSTEM VALUE SELECT "CUSTOMER_ID", "COMPANY_NAME", "CONTACT_NAME", "CONTACT_TITLE", "ADDRESS", "CITY", "REGION", "POSTAL_CODE", "COUNTRY", "PHONE", "FAX" FROM "PUBLIC"."CUSTOMERS" [23502-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:365)
	at org.h2.table.Table.convertInsertRow(Table.java:936)
	at org.h2.command.dml.Insert.addRow(Insert.java:229)
	at org.h2.command.dml.Insert.insertRows(Insert.java:203)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:649)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:561)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:343)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:244)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:413)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.executeUpdate(CommandRemote.java:237)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:262)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:78)
	... 24 more
[ WARN] 2024-04-10 12:37:12,307 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 42122, SQLState: 42S22
[ERROR] 2024-04-10 12:37:12,307 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
[ WARN] 2024-04-10 12:37:12,311 com.targetindia.miniproject.dao.JpaCustomerDao error while calling JpaCustomerDao.getAll()
org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[ WARN] 2024-04-10 12:37:12,312 com.targetindia.miniproject.ui.Main there was an error while calling cm.getAll()
com.targetindia.miniproject.service.ServiceException: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:35)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:64)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	... 3 more
Caused by: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	... 4 more
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[TRACE] 2024-04-10 12:38:13,901 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-10 12:38:13,903 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 42122, SQLState: 42S22
[ERROR] 2024-04-10 12:38:13,903 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
[ WARN] 2024-04-10 12:38:13,904 com.targetindia.miniproject.dao.JpaCustomerDao error while calling JpaCustomerDao.getAll()
org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[ WARN] 2024-04-10 12:38:13,907 com.targetindia.miniproject.ui.Main there was an error while calling cm.getAll()
com.targetindia.miniproject.service.ServiceException: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:35)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:64)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	... 3 more
Caused by: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	... 4 more
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[TRACE] 2024-04-10 12:52:06,837 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-10 12:52:09,603 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-10 12:52:09,920 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[ WARN] 2024-04-10 12:52:10,623 org.hibernate.tool.schema.internal.ExceptionHandlerLoggedImpl GenerationTarget encountered exception accepting command : Error executing DDL "
    alter table if exists shippers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "
    alter table if exists shippers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:92)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlString(AbstractSchemaMigrator.java:574)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlStrings(AbstractSchemaMigrator.java:514)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.migrateTable(AbstractSchemaMigrator.java:333)
	at org.hibernate.tool.schema.internal.GroupedSchemaMigratorImpl.performTablesMigration(GroupedSchemaMigratorImpl.java:84)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.performMigration(AbstractSchemaMigrator.java:232)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:117)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:284)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:143)
	at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:140)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:37)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:292)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:431)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1455)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at com.targetindia.miniproject.utils.JpaUtil.<clinit>(JpaUtil.java:14)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:59)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: NULL not allowed for column "ID"; SQL statement:
INSERT INTO "PUBLIC"."SHIPPERS_COPY_6_7"("SHIPPER_ID", "COMPANY_NAME", "PHONE") OVERRIDING SYSTEM VALUE SELECT "SHIPPER_ID", "COMPANY_NAME", "PHONE" FROM "PUBLIC"."SHIPPERS" [23502-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:365)
	at org.h2.table.Table.convertInsertRow(Table.java:936)
	at org.h2.command.dml.Insert.addRow(Insert.java:229)
	at org.h2.command.dml.Insert.insertRows(Insert.java:203)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:649)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:561)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:343)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:244)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:413)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.executeUpdate(CommandRemote.java:237)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:262)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:78)
	... 24 more
[ WARN] 2024-04-10 12:52:10,807 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 42122, SQLState: 42S22
[ERROR] 2024-04-10 12:52:10,807 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]
[ WARN] 2024-04-10 12:52:10,810 com.targetindia.miniproject.dao.JpaCustomerDao error while calling JpaCustomerDao.getAll()
org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[ WARN] 2024-04-10 12:52:10,811 com.targetindia.miniproject.ui.Main there was an error while calling cm.getAll()
com.targetindia.miniproject.service.ServiceException: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0]
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:35)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0]
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:64)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	... 3 more
Caused by: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	... 4 more
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[TRACE] 2024-04-10 12:55:42,741 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-10 12:55:44,084 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-10 12:55:44,401 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[ WARN] 2024-04-10 12:55:45,112 org.hibernate.tool.schema.internal.ExceptionHandlerLoggedImpl GenerationTarget encountered exception accepting command : Error executing DDL "
    alter table if exists shippers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "
    alter table if exists shippers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:92)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlString(AbstractSchemaMigrator.java:574)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlStrings(AbstractSchemaMigrator.java:514)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.migrateTable(AbstractSchemaMigrator.java:333)
	at org.hibernate.tool.schema.internal.GroupedSchemaMigratorImpl.performTablesMigration(GroupedSchemaMigratorImpl.java:84)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.performMigration(AbstractSchemaMigrator.java:232)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:117)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:284)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:143)
	at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:140)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:37)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:292)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:431)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1455)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at com.targetindia.miniproject.utils.JpaUtil.<clinit>(JpaUtil.java:14)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:59)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: NULL not allowed for column "ID"; SQL statement:
INSERT INTO "PUBLIC"."SHIPPERS_COPY_7_16"("SHIPPER_ID", "COMPANY_NAME", "PHONE", "CITY", "EMAIL", "NAME") OVERRIDING SYSTEM VALUE SELECT "SHIPPER_ID", "COMPANY_NAME", "PHONE", "CITY", "EMAIL", "NAME" FROM "PUBLIC"."SHIPPERS" [23502-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:365)
	at org.h2.table.Table.convertInsertRow(Table.java:936)
	at org.h2.command.dml.Insert.addRow(Insert.java:229)
	at org.h2.command.dml.Insert.insertRows(Insert.java:203)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:649)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:561)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:343)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:244)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:413)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.executeUpdate(CommandRemote.java:237)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:262)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:78)
	... 24 more
[ WARN] 2024-04-10 12:55:45,288 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 42122, SQLState: 42S22
[ERROR] 2024-04-10 12:55:45,288 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]
[ WARN] 2024-04-10 12:55:45,291 com.targetindia.miniproject.dao.JpaCustomerDao error while calling JpaCustomerDao.getAll()
org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[ WARN] 2024-04-10 12:55:45,292 com.targetindia.miniproject.ui.Main there was an error while calling cm.getAll()
com.targetindia.miniproject.service.ServiceException: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0]
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:35)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0]
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:64)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	... 3 more
Caused by: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	... 4 more
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from shippers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[TRACE] 2024-04-10 12:57:21,324 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-10 12:57:22,810 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-10 12:57:23,132 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[ WARN] 2024-04-10 12:57:23,850 org.hibernate.tool.schema.internal.ExceptionHandlerLoggedImpl GenerationTarget encountered exception accepting command : Error executing DDL "
    alter table if exists customers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "
    alter table if exists customers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:92)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlString(AbstractSchemaMigrator.java:574)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlStrings(AbstractSchemaMigrator.java:514)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.migrateTable(AbstractSchemaMigrator.java:333)
	at org.hibernate.tool.schema.internal.GroupedSchemaMigratorImpl.performTablesMigration(GroupedSchemaMigratorImpl.java:84)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.performMigration(AbstractSchemaMigrator.java:232)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:117)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:284)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:143)
	at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:140)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:37)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:292)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:431)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1455)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at com.targetindia.miniproject.utils.JpaUtil.<clinit>(JpaUtil.java:14)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:59)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: NULL not allowed for column "ID"; SQL statement:
INSERT INTO "PUBLIC"."CUSTOMERS_COPY_8_17"("CUSTOMER_ID", "COMPANY_NAME", "CONTACT_NAME", "CONTACT_TITLE", "ADDRESS", "CITY", "REGION", "POSTAL_CODE", "COUNTRY", "PHONE", "FAX", "EMAIL", "NAME") OVERRIDING SYSTEM VALUE SELECT "CUSTOMER_ID", "COMPANY_NAME", "CONTACT_NAME", "CONTACT_TITLE", "ADDRESS", "CITY", "REGION", "POSTAL_CODE", "COUNTRY", "PHONE", "FAX", "EMAIL", "NAME" FROM "PUBLIC"."CUSTOMERS" [23502-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:365)
	at org.h2.table.Table.convertInsertRow(Table.java:936)
	at org.h2.command.dml.Insert.addRow(Insert.java:229)
	at org.h2.command.dml.Insert.insertRows(Insert.java:203)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:649)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:561)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:343)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:244)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:413)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.executeUpdate(CommandRemote.java:237)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:262)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:78)
	... 24 more
[ WARN] 2024-04-10 12:57:24,024 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 42122, SQLState: 42S22
[ERROR] 2024-04-10 12:57:24,024 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
[ WARN] 2024-04-10 12:57:24,027 com.targetindia.miniproject.dao.JpaCustomerDao error while calling JpaCustomerDao.getAll()
org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[ WARN] 2024-04-10 12:57:24,028 com.targetindia.miniproject.ui.Main there was an error while calling cm.getAll()
com.targetindia.miniproject.service.ServiceException: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:35)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:64)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	... 3 more
Caused by: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	... 4 more
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[TRACE] 2024-04-10 12:57:53,735 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-10 12:57:55,767 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-10 12:57:56,083 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[ WARN] 2024-04-10 12:57:56,790 org.hibernate.tool.schema.internal.ExceptionHandlerLoggedImpl GenerationTarget encountered exception accepting command : Error executing DDL "
    alter table if exists customers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "
    alter table if exists customers 
       add column id integer not null" via JDBC [NULL not allowed for column "ID";]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:92)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlString(AbstractSchemaMigrator.java:574)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlStrings(AbstractSchemaMigrator.java:514)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.migrateTable(AbstractSchemaMigrator.java:333)
	at org.hibernate.tool.schema.internal.GroupedSchemaMigratorImpl.performTablesMigration(GroupedSchemaMigratorImpl.java:84)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.performMigration(AbstractSchemaMigrator.java:232)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:117)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:284)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:143)
	at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:140)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:37)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:292)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:431)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1455)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at com.targetindia.miniproject.utils.JpaUtil.<clinit>(JpaUtil.java:14)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:59)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: NULL not allowed for column "ID"; SQL statement:
INSERT INTO "PUBLIC"."CUSTOMERS_COPY_9_18"("CUSTOMER_ID", "COMPANY_NAME", "CONTACT_NAME", "CONTACT_TITLE", "ADDRESS", "CITY", "REGION", "POSTAL_CODE", "COUNTRY", "PHONE", "FAX", "EMAIL", "NAME") OVERRIDING SYSTEM VALUE SELECT "CUSTOMER_ID", "COMPANY_NAME", "CONTACT_NAME", "CONTACT_TITLE", "ADDRESS", "CITY", "REGION", "POSTAL_CODE", "COUNTRY", "PHONE", "FAX", "EMAIL", "NAME" FROM "PUBLIC"."CUSTOMERS" [23502-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:365)
	at org.h2.table.Table.convertInsertRow(Table.java:936)
	at org.h2.command.dml.Insert.addRow(Insert.java:229)
	at org.h2.command.dml.Insert.insertRows(Insert.java:203)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:649)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:561)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:343)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:244)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:413)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:520)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.executeUpdate(CommandRemote.java:237)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:262)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:78)
	... 24 more
[ WARN] 2024-04-10 12:57:56,965 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 42122, SQLState: 42S22
[ERROR] 2024-04-10 12:57:56,965 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
[ WARN] 2024-04-10 12:57:56,969 com.targetindia.miniproject.dao.JpaCustomerDao error while calling JpaCustomerDao.getAll()
org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[ WARN] 2024-04-10 12:57:56,970 com.targetindia.miniproject.ui.Main there was an error while calling cm.getAll()
com.targetindia.miniproject.service.ServiceException: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:35)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:64)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	... 3 more
Caused by: org.hibernate.exception.SQLGrammarException: could not prepare statement [Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]] [select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:64)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:187)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:76)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.lambda$list$0(JdbcSelectExecutorStandardImpl.java:102)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:226)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:163)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:204)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	... 4 more
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "C1_0.ID" not found; SQL statement:
select c1_0.id,c1_0.city,c1_0.email,c1_0.name,c1_0.phone from customers c1_0 [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:289)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.prepare(CommandRemote.java:78)
	at org.h2.command.CommandRemote.<init>(CommandRemote.java:50)
	at org.h2.engine.SessionRemote.prepareCommand(SessionRemote.java:487)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:93)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:316)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$1.doPrepare(StatementPreparerImpl.java:91)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:177)
	... 25 more
[TRACE] 2024-04-10 14:59:20,449 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-11 16:00:51,585 com.targetindia.miniproject.ui.Main starting the app...
[TRACE] 2024-04-11 16:00:53,061 com.targetindia.miniproject.ui.Main user made a choice 1
[ WARN] 2024-04-11 16:00:53,518 org.hibernate.orm.connections.pooling HHH10001002: Using built-in connection pool (not intended for production use)
[ WARN] 2024-04-11 16:00:54,393 org.hibernate.tool.schema.internal.ExceptionHandlerLoggedImpl GenerationTarget encountered exception accepting command : Error executing DDL "
    alter table if exists customers 
       alter column CUSTOMER_ID set data type integer" via JDBC [Data conversion error converting "'ALFKI' (CUSTOMERS_COPY_4_0: ""CUSTOMER_ID"" INTEGER NOT NULL)";]
org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "
    alter table if exists customers 
       alter column CUSTOMER_ID set data type integer" via JDBC [Data conversion error converting "'ALFKI' (CUSTOMERS_COPY_4_0: ""CUSTOMER_ID"" INTEGER NOT NULL)";]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:92)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlString(AbstractSchemaMigrator.java:574)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlStrings(AbstractSchemaMigrator.java:514)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.migrateTable(AbstractSchemaMigrator.java:333)
	at org.hibernate.tool.schema.internal.GroupedSchemaMigratorImpl.performTablesMigration(GroupedSchemaMigratorImpl.java:84)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.performMigration(AbstractSchemaMigrator.java:232)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:117)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:284)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:143)
	at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:140)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:37)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:292)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:431)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1455)
	at org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at com.targetindia.miniproject.utils.JpaUtil.<clinit>(JpaUtil.java:14)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:59)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'ALFKI' (CUSTOMERS_COPY_4_0: ""CUSTOMER_ID"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CUSTOMERS_COPY_4_0"("CUSTOMER_ID", "COMPANY_NAME", "CONTACT_NAME", "CONTACT_TITLE", "ADDRESS", "CITY", "REGION", "POSTAL_CODE", "COUNTRY", "PHONE", "FAX", "EMAIL", "NAME") OVERRIDING SYSTEM VALUE SELECT "CUSTOMER_ID", "COMPANY_NAME", "CONTACT_NAME", "CONTACT_TITLE", "ADDRESS", "CITY", "REGION", "POSTAL_CODE", "COUNTRY", "PHONE", "FAX", "EMAIL", "NAME" FROM "PUBLIC"."CUSTOMERS" [22018-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:518)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.table.Column.getDataConversionError(Column.java:413)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:372)
	at org.h2.table.Table.convertInsertRow(Table.java:936)
	at org.h2.command.dml.Insert.addRow(Insert.java:229)
	at org.h2.command.dml.Insert.insertRows(Insert.java:203)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:649)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:561)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:343)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:229)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.server.TcpServerThread.process(TcpServerThread.java:413)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:191)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.message.DbException: Data conversion error converting "ALFKI" [22018-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueStringBase.getInt(ValueStringBase.java:133)
	at org.h2.value.Value.convertToInt(Value.java:1596)
	at org.h2.value.Value.convertTo(Value.java:1156)
	at org.h2.value.Value.convertForAssignTo(Value.java:1113)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	... 16 more
Caused by: org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "ALFKI" [22018-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:518)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 22 more
Caused by: java.lang.NumberFormatException: For input string: "ALFKI"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at org.h2.value.ValueStringBase.getInt(ValueStringBase.java:131)
	... 20 more

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:518)
	at org.h2.engine.SessionRemote.readException(SessionRemote.java:650)
	at org.h2.engine.SessionRemote.done(SessionRemote.java:619)
	at org.h2.command.CommandRemote.executeUpdate(CommandRemote.java:237)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:262)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:78)
	... 24 more
[ WARN] 2024-04-11 16:00:54,586 org.hibernate.engine.jdbc.spi.SqlExceptionHelper SQL Error: 22018, SQLState: 22018
[ERROR] 2024-04-11 16:00:54,586 org.hibernate.engine.jdbc.spi.SqlExceptionHelper Data conversion error converting "ALFKI" [22018-224]
[ WARN] 2024-04-11 16:00:54,594 com.targetindia.miniproject.dao.JpaCustomerDao error while calling JpaCustomerDao.getAll()
org.hibernate.exception.DataException: Could not extract column [1] from JDBC ResultSet [Data conversion error converting "ALFKI" [22018-224]] [n/a]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:53)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:94)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.readCurrentRowValues(JdbcValuesResultSetImpl.java:270)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advance(JdbcValuesResultSetImpl.java:243)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "ALFKI" [22018-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:518)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueStringBase.getInt(ValueStringBase.java:133)
	at org.h2.jdbc.JdbcResultSet.getIntInternal(JdbcResultSet.java:348)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:319)
	at org.hibernate.type.descriptor.jdbc.IntegerJdbcType$2.doExtract(IntegerJdbcType.java:88)
	at org.hibernate.type.descriptor.jdbc.BasicExtractor.extract(BasicExtractor.java:44)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.readCurrentRowValues(JdbcValuesResultSetImpl.java:262)
	... 21 more
Caused by: java.lang.NumberFormatException: For input string: "ALFKI"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:665)
	at java.base/java.lang.Integer.parseInt(Integer.java:781)
	at org.h2.value.ValueStringBase.getInt(ValueStringBase.java:131)
	... 26 more
[ WARN] 2024-04-11 16:00:54,598 com.targetindia.miniproject.ui.Main there was an error while calling cm.getAll()
com.targetindia.miniproject.service.ServiceException: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.DataException: Could not extract column [1] from JDBC ResultSet [Data conversion error converting "ALFKI" [22018-224]] [n/a]
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:35)
	at com.targetindia.miniproject.ui.Main.displayCustomerList(Main.java:180)
	at com.targetindia.miniproject.ui.Main.start(Main.java:54)
	at com.targetindia.miniproject.ui.Main.main(Main.java:202)
Caused by: com.targetindia.miniproject.dao.DaoException: org.hibernate.exception.DataException: Could not extract column [1] from JDBC ResultSet [Data conversion error converting "ALFKI" [22018-224]] [n/a]
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:64)
	at com.targetindia.miniproject.service.CustomerManager.getAll(CustomerManager.java:32)
	... 3 more
Caused by: org.hibernate.exception.DataException: Could not extract column [1] from JDBC ResultSet [Data conversion error converting "ALFKI" [22018-224]] [n/a]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:53)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:56)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:94)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.readCurrentRowValues(JdbcValuesResultSetImpl.java:270)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advance(JdbcValuesResultSetImpl.java:243)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:84)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:29)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:66)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:198)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:361)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:168)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.list(JdbcSelectExecutorStandardImpl.java:93)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:31)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$0(ConcreteSqmSelectQueryPlan.java:110)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:303)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:244)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:518)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:367)
	at org.hibernate.query.Query.getResultList(Query.java:119)
	at com.targetindia.miniproject.dao.JpaCustomerDao.getAll(JpaCustomerDao.java:61)
	... 4 more
Caused by: org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "ALFKI" [22018-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:518)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueStringBase.getInt(ValueStringBase.java:133)
	at org.h2.jdbc.JdbcResultSet.getIntInternal(JdbcResultSet.java:348)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:319)
	at org.hibernate.type.descriptor.jdbc.IntegerJdbcType$2.doExtract(IntegerJdbcType.java:88)
	at org.hibernate.type.descriptor.jdbc.BasicExtractor.extract(BasicExtractor.java:44)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.readCurrentRowValues(JdbcValuesResultSetImpl.java:262)
	... 21 more
Caused by: java.lang.NumberFormatException: For input string: "ALFKI"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:665)
	at java.base/java.lang.Integer.parseInt(Integer.java:781)
	at org.h2.value.ValueStringBase.getInt(ValueStringBase.java:131)
	... 26 more
